Gemfile

```diff
@@ -3,14 +3,14 @@ source 'http://rubygems.org'
 ruby '2.2.3'
 
 group :development do
-  gem 'activesupport'
-  gem 'cucumber'
-  gem 'cucumber_lint'
-  gem 'kappamaki'
-  gem 'mortadella'
-  gem 'open4'
-  gem 'parallel_tests'
-  gem 'rake'
-  gem 'rubocop'
-  gem 'rspec'
+  gem 'activesupport', '4.2.6'
```

Comment:
In my opinion Bundler has solved this the best way, and Node is an inferior and broken solution in regards to versioning. Usually the particular versions of gems don't matter much, except when updating. And then you can see the changes in the diff of Gemfile.lock. In this case we want to hard-code the version of Cucumber, so let's do that here, and leave the other gems open. Let's stay idiomatic here.